src/data/Pet.js

```diff
@@ -19,10 +23,42 @@ export class Pet {
         this.hunger = 100;
         this.state = STATES.IDLE;
         this.thought = THOUGHTS.HAPPY;
-        this.poop = true;
+        this.poop = false;
+        this.poopCountdown = COUNTDOWNS.POOP;
+    }
+
+    /**
+     * Called when the pet has had its poop cleaned. Sets poop to false to hide the icon.
+     */
+    cleanedPoop(){
+        this.poop = false;
     }
 
-    // ADD FUNCTIONS HERE TO AFFECT PET STATS
+    /**
+     * Called every second. Calls helper methods that update the pet after a certain number of seconds
+     * Returns true if the pet needs updated.
+     */
+    updatePet(){
+        if (this.poopCooldown()){   //Can just add methods to this if statement
```

Comment:
I like this approach

Reply:
we could also just return the evaluation of the condition (like `return (this.poopCooldown())` but this may be easier to read